packages/components/src/components/carousel/carousel.stories.ts

```diff
@@ -137,6 +137,28 @@ export const SlidesPerPage = {
   }
 };
 
+/**
+ * Use `slides-per-move` to set how many slides the carousel advances when scrolling. Useful when specifying a `slides-per-page`
```

Comment:
what happens when `slides-per-page` is set to 1 and `slides-per-move` is set to 2? maybe you can explain the combination a bit more detailed in terms of what to expect => makes it easier to test.

Reply:
@karlbaumhauer I added some explanation there, but I thought it was better to show how it works so I expanded on it's story to show that specific example.